app/scripts/views/mixins/signup-mixin.js

```diff
@@ -31,6 +31,8 @@ define(function (require, exports, module) {
      */
     signUp: function (account, password) {
       var self = this;
+      self.logEvent('flow.attempt_signup');
```

Comment:
Does this correspond to `signup.submit` from the [spec](https://github.com/mozilla/fxa/tree/master/features/FxA-41-signin-funnel-metrics#detailed-design)? Is there a reason to prefer this name over that one?